test/commands/cbd/cbd.spec.ts

```diff
@@ -31,7 +32,7 @@ describe('sasjs cbd (global config)', () => {
       name: targetName,
       serverType,
       serverUrl: process.env.SERVER_URL as string,
-      appLoc: `/Public/app/cli-tests/${targetName}-${timestamp}`,
+      appLoc: `/Public/app/cli-tests/${targetName}`,
```

Comment:
is it safe to remove the timestamp, given we are running tests in parallel?

Reply:
aah - ignore - we are adding the timestamp to the `targetName`, makes sense

Reply:
Change made to `targetName`, so that targets in global config should not clash, in any case.